src/routes/about.svelte

```diff
@@ -178,28 +178,56 @@
   </h3>
   <ul class="mb-3">
     <li>
-      {$_('gl.as', { default: 'Assamese' })}: Palash Nath, Luke Horo
+      <b>{$_('gl.as', { default: 'Assamese' })}</b>: Palash Nath, Luke Horo, Kapil Medhi, Dr. Seuji
+      Sharma, Dr. Gitanjali Bezbaruah, Biren Baruah, Khagendra Nath Medhi, Pranab Sharma, Dhanmani
+      Baishya Student, Chan Mohammad Ali, Rahul Choudhary
     </li>
-    <li>{$_('gl.bn', { default: 'Bangla' })}: Sumedha Sengupta, Prof. Arun Ghosh</li>
-    <li>{$_('gl.fr', { default: 'French' })}: Anna Luisa Daigneault</li>
     <li>
-      {$_('gl.he', { default: 'Hebrew' })}: Dana Melaver, Daniel Bögre Udell
+      <span class="font-semibold">{$_('gl.bn', { default: 'Bangla' })}</span>: Sumedha Sengupta,
```

Comment:
Great! I like the look. How about making them all bold with just the <b> tag as you did for Assamese? Less code and though it's slightly bolder than 'font-semibold' I think both are good.

Reply:
Oh, it was a good accident then. I thought I had changed everything.

The tag bold has semantic meaning, If you think is still good, I can do it with the tag instead :)

Reply:
Yes, please use the < b > tag.